docs/components/modeler/web-modeler/process-application-pipeline.md

```diff
@@ -0,0 +1,92 @@
+
+:::caution
+For business-critical and higher-risk processes that require strict governance and/or quality requirements, you can [integrate Web Modeler into your CI/CD pipelines](/docs/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd.md).
+:::
+
+## Model
+
+During the modeling stage, you will typically:
+
+- [Create a process application](create-a-process-application.md) and select a default development cluster to deploy to.
+- Invite other users to collaborate on the process application.
+- Define and setup the clusters and deployment stages that you will use in your development pipeline.
+- Model your diagrams and any associated resources, and fix any errors shown in the modeler.
+
+### Deployment stages
+
+You can deploy and promote an application process version between the stages of a standard development life cycle (development, testing, staging, production).
+
+To define your pipeline cluster stages:
+
+1. Open the [main process](create-a-process-application.md#main-process).
+1. Select **Deploy** to open the **Deploy process application** modal.
+1. Select **Define stages** to open the **Define stages** modal.
+   <p><img src={DeployStagesImg} alt="Define stage for deployment modal" /></p>
+1. Select and assign a cluster to each deployment stage that you want to use in your pipeline.
+1. Select **Save** to save your changes and close the modal.
+
+:::note
+
+- An administrator should typically define the stages and clusters within Web Modeler.
+- During deployment, the next stage is not automatically selected. You must choose the stage you want to promote to.
+- You must select a cluster for at least one stage to be able to deploy. An **Undefined stages** warning is shown if no cluster is selected for at least one stage.
+
+:::
+
+You can define and deploy to the following stages:
+
+| Stage       | Description                                                                                                     |
+| :---------- | :-------------------------------------------------------------------------------------------------------------- |
+| Development | Use to create and test new software features and changes.                                                       |
+| Testing     | Use for quality checks, ensuring software meets defined standards before release.                               |
+| Staging     | Use for controlled testing where changes are validated before deployment to production.                         |
+| Production  | The live system with the latest software. Only administrators and organization owners can deploy to this stage. |
+
+## Validate
+
+When your process application is ready for validation you can deploy it to your development cluster.
+
+- Use [Play mode](play-your-process.md) to quickly validate the process behavior and play different scenarios.
+- Use [token simulation](token-simulation.md) to correct and optimize your process flow.
```

Comment:
Token simulation is most valuable during the design phase because it ignores implementation details. By the time it's implemented, the user should use Play

Reply:
Moved to the Model stage 👍